spec/Modules.tex

```diff
@@ -220,40 +226,57 @@ \section{Access of Module Contents}
+via the use statement~(\rsec{Using_Modules}) or qualified
+naming~(\rsec{Explicit_Naming}).
 
 \subsection{Visibility Of A Module}
 \label{Visibility_Of_A_Module}
 \index{modules!access}
 
-A module defined at file scope is visible anywhere. The visibility of a nested
-module is subject to the rules of~\rsec{Visibility_Of_Symbols}. There,
-the nested module is considered a "symbol defined at the top level
-scope" of its outer module.
+A top-level is available for use~(\rsec{Using_Modules}) anywhere. The
+visibility of a nested module is subject to the rules
+of~\rsec{Visibility_Of_Symbols}, where the nested module is considered
+a "module-scope symbol" of its outer module.
 
 \subsection{Visibility Of A Module's Symbols}
 \label{Visibility_Of_Symbols}
 \index{modules!access}
 
-A symbol defined at the top level scope of a module is \emph{visible}
-from outside the module when the \sntx{privacy-specifier} of its
-definition is \chpl{public} or is omitted (i.e. by default). When a
-symbol defined at the top level scope of a module is declared
+A symbol defined at module scope is \emph{visible} from outside the
+module when the \sntx{privacy-specifier} of its definition
+is \chpl{public} or is omitted (i.e. by default). When a module-scope
+symbol is declared
 \chpl{private}, it is not visible outside of that module. A
 symbol's visibility inside its module is controlled by normal lexical
-scoping and is not affected by its \sntx{privacy-specifier}.  A
-module's visible symbols are accessible via explicit
-naming~(\rsec{Explicit_Naming}) or the use
-statement~(\rsec{Using_Modules}) only where the module's symbol is
-visible~(\rsec{Visibility_Of_A_Module}).
+scoping and is not affected by its \sntx{privacy-specifier}.  When a
+module's symbol is visible~(\rsec{Visibility_Of_A_Module}), the
+symbols it contains are accessible via module's visible symbols are
+accessible via the use statement~(\rsec{Using_Modules}) or qualified
+naming~(\rsec{Explicit_Naming}).
+
+\subsection{Using Modules}
+\label{Using_Modules}
+\index{modules!using}
 
-\subsection{Explicit Naming}
+The \chpl{use} statement provides the primary means of accessing a
+module's symbols from outside of the module.  Use statements make both
+the module's name and its public symbols available for reference
+within a given scope.  For top-level modules, a use statement is
+required before referring to the module's name or the symbols it
+contains within a given lexical scope.
+
+Use statements can also restrict or rename the set of module symbols
+that are available within the scope.  For further information about
+use statements, see~\rsec{The_Use_Statement}.
+
+
+\subsection{Qualified Naming of Module Symbols}
 \label{Explicit_Naming}
-\index{modules!explicitly named}
+\index{modules!qualified naming}
 
-All publicly visible top-level module symbols can be named explicitly
-with the following syntax:
+When a module's symbol is visible—via a use statement, or lexically
```

Comment:
Dunno if this renders right already, but I'd find it easier to modify in the future if there were spaces between the hyphen, e.g. "visible - via" instead of "visible-via"

Reply:
Oops, that's an M-dash, though it's hard to tell when rendered as text.  I'll switch to a LaTeX M-dash (`---`) for clarity.  The style guides I've typically used prefer not to put spaces between m-dashes and the adjacent words (the m-dash already takes up a lot of space).